docs/blog/posts/2025-04-14-incident-report-organization-team-privileges.md

```diff
@@ -0,0 +1,223 @@
+ holygrail  | 2023-07-06 12:55:43.261593 | organization:organization_role:remove | blackknight
+ ni         | 2023-09-18 12:07:17.389244 | organization:organization_role:remove | shrubbery
+ parrot     | 2024-02-04 19:23:25.354344 | organization:organization_role:remove | exparrot
+ spam       | 2024-08-24 01:40:22.405746 | organization:organization_role:remove | spamlover
+ spam       | 2025-02-09 18:14:13.891224 | organization:team_role:remove         | eggandspam
+ albatross  | 2025-03-07 06:55:29.446617 | organization:organization_role:remove | nudge
+ albatross  | 2025-03-07 06:55:37.271176 | organization:organization_role:remove | wink
+ cheese     | 2025-03-13 18:25:54.650905 | organization:team_role:remove         | gorgonzola
+ cheese     | 2025-03-13 18:26:02.525162 | organization:team_role:remove         | camembert
+ ministry   | 2025-03-20 07:53:45.616404 | organization:organization_role:remove | sillywalks
+ argument   | 2025-03-31 15:52:18.186223 | organization:organization_role:remove | contradiction
+ fishslap   | 2025-04-14 15:12:14.023183 | organization:organization_role:remove | danceking
+ fishslap   | 2025-04-14 15:24:54.208641 | organization:organization_role:remove | danceking
+ fishslap   | 2025-04-14 15:27:22.954624 | organization:team_role:remove         | danceking
+```
+
+Here, we see the `spamlover` user being removed from the `spam` Organization
+on `2024-08-24`, without being removed from the team, confirming our finding from the
+earlier query.
+
+We also see the User `danceking` from the `fishslap` Organization being removed from
+the Organization multiple times, before the reporter removed them from their assigned
+Team.
+
+This allowed us to confirm that beyond the already identified incidents,
+no other Organizations had found this problem before without letting us know.
+
+## Thanks
+
+First and foremost, thanks to our reporter for finding and reporting this issue.
+
+TODO: Credit? awaiting response from reporter
+
+We are grateful for the entire community of security researchers and users who
+find and report security issues to PyPI in accordance with our
+[Security Policy](https://pypi.org/security/).
+PyPI relies on the efforts of our community to help us find and resolve issues like
+these before they become critical issues.
+Cooperation between all parties helps to improve the security of open source,
+and none of us could do it alone.
+
+The tools and capabilities we've evolved in PyPI over the past six years have really
+come to be an asset in situations like these. I'm grateful to all the contributors
+and admins who have helped us to build them 💜.
+
+---
+
+_Ee Durbin is the Director of Infrastructure at
+the Python Software Foundation.
+They have been contributing to keeping PyPI online, available, and
+secure since 2013._
+
+[^1]:
+    As of writing, there are 6,682 remainging Organization Requests to review.
```

Comment:
typo: remaining*

Reply:
already resolved in https://github.com/pypi/warehouse/pull/17958/commits/1e1d5aff92c44e69ea580e55f136690ea6db9cdc